sdks/java/extensions/sketching/src/main/java/org/apache/beam/sdk/extensions/sketching/frequency/SketchFrequencies.java

```diff
@@ -0,0 +1,357 @@
+      }
+      return new CountMinSketchFn(width, depth, this.seed);
+    }
+
+    /**
+     * Returns an {@code CountMinSketchFn} combiner that will be as accurate as specified. The
+     * relative error {@code epsilon} can be guaranteed only with a certain {@code confidence},
+     * which has to be between 0 and 1 (1 being of course impossible). Those parameters will
+     * determine the size of the Count-min sketch in which the elements will be aggregated.
+     * This method can only be applied to a {@link CountMinSketchFn} already created with the
+     * method {@link CountMinSketchFn#create(int)}.
+     *
+     * <p>The lower the {@code epsilon} value, the greater the width.
+     * <pre>{@code width = (int) 2 / epsilon)}</pre>
+     *
+     * <p>The greater the confidence, the greater the depth.
+     * <pre>{@code depth = (int) -log2(1 - confidence)}</pre>
+     *
+     * <p>Example of use:
+     * <br>1) Globally :
+     * <pre>{@code
+     * PCollection<String> pc = ...;
+     * PCollection<CountMinSketch> countMinSketch =
+     *     pc.apply(Combine.globally(CountMinSketchFn.<String>create(1234)
+     *                  withDimensions(0.001, 0.99));
+     * }</pre>
+     * <br>2) Per key :
+     * <pre>{@code
+     * PCollection<KV<Integer, String>> pc = ...;
+     * PCollection<KV<Integer, CountMinSketch>> countMinSketch =
+     *     pc.apply(Combine.perKey(CountMinSketchFn.<String>create(1234)
+     *                  withAccuracy(0.001, 0.99)););
+     * }</pre>
+     *
+     *
+     * @param epsilon the relative error of the result
+     * @param confidence the confidence in the result to not overcome the relative error
+     */
+    public CountMinSketchFn withAccuracy(double epsilon, double confidence) {
+      return new CountMinSketchFn(epsilon, confidence, this.seed);
+    }
+
+    @Override public CountMinSketch createAccumulator() {
+      return new CountMinSketch(this.depth, this.width, this.seed);
+    }
+
+    @Override public CountMinSketch addInput(CountMinSketch accumulator, String element) {
+      accumulator.add(element, 1);
+      return accumulator;
+    }
+
+    @Override public CountMinSketch mergeAccumulators(Iterable<CountMinSketch> accumulators) {
+      Iterator<CountMinSketch> it = accumulators.iterator();
+      if (!it.hasNext()) {
+        return new CountMinSketch(seed, width, depth);
+      }
+      CountMinSketch merged = it.next();
+      try {
+        while (it.hasNext()) {
+          merged = CountMinSketch.merge(merged, it.next());
+        }
+      } catch (FrequencyMergeException e) {
+        // Should never happen because all the accumulators created are of the same type.
+        LOG.error(e.getMessage(), e);
+      }
+      return merged;
+    }
+
+    @Override public CountMinSketch extractOutput(CountMinSketch accumulator) {
+      return accumulator;
+    }
+
+    @Override public Coder<CountMinSketch> getAccumulatorCoder(CoderRegistry registry,
```

Comment:
These two functions for returning coders are unnecessary if instead you provide a default coder for `CountMinSketch` globally, per https://beam.apache.org/contribute/ptransform-style-guide/#providing-default-coders-for-types.